perl/CHANGES

```diff
@@ -0,0 +1,168 @@
+Revision history for perl module Gherkin.pm, and the upstream Gherkin project. This file is automatically generated, don't edit by hand. Instead edit the CPAN-ONLY-CHANGES file in the git repo.
```

Comment:
I'd rather not have to maintain build scripts to create a modified copy CHANGES for every implementation. It's also harder for developers to have many CHANGES files to maintain. Can we remove this file?